src/komodo_hardfork.cpp

```diff
@@ -17,8 +17,8 @@ const int32_t nS6HardforkHeight = 2963330;  // dPoW Season 6, Fri Jun 24 2022
 const uint32_t nS7Timestamp = 1688132253;   // dPoW Season 7, Fri Jun 30 2023 13:37:33 GMT+0000
 const int32_t nS7HardforkHeight = 3484958;  // dPoW Season 7, Fri Jun 30 2023
 
-const uint32_t nS8Timestamp = 1726321053;   // dPoW Season 8, Sat Sep 14 2024 13:37:33 GMT+0000
-const int32_t nS8HardforkHeight = 4100333;  // dPoW Season 8, Sat Sep 14 2024
+const uint32_t nS8Timestamp = 1728049053;   // dPoW Season 8, Fri Oct 4 2024 13:37:33 GMT+0000
+const int32_t nS8HardforkHeight = 4125988;  // dPoW Season 8, Fri Oct 4 2024 (est. via https://github.com/KomodoPlatform/NotaryNodes/blob/master/utils/estimate_hf_block.py)
```

Comment:
current estimation for height `4125988` is October 1st

cc @DeckerSU @smk762

Reply:
Ideally this should be (re)calculated as late as possible to make it as accurate as possible wrt to recent/anticipated block production rate.

Reply:
ok, will recalc tonight and include in https://github.com/KomodoPlatform/komodo/pull/623

Reply:
I created my own script for these calculations, which you can find here: [estimate-blocks.py](https://github.com/DeckerSU/komodo_scripts/blob/master/estimate-blocks.py). According to my calculations, we have the following:
```
- Average blocks per day: 1370.96
- Minimum blocks per day: 1260
- Maximum blocks per day: 1457
- Current height: 4063104
- Days to future: 45
- Estimated block height at 2024-10-04 13:37:33: 4124797
```
Based on this, `October 4th` corresponds to approximately block `4124797` on average. Currently, we have `4125988` hardcoded, which should occur later than October 4th. So, my guess is that `4125988` is okay.

Reply:
thanks for validating @DeckerSU - lets keep the height as is then